fix `gtest` deprecation warnings

Check if the change is compatible with `vt`'s bundled `gtest` (ver. `1.12.1`) and fix the deprecation warning:

```
/checkpoint/tests/unit/test_kokkos_serialize_0d.cc:51:1: warning: 'TypedTestCase_P_IsDeprecated' is deprecated: TYPED_TEST_CASE_P is deprecated, please use TYPED_TEST_SUITE_P [-Wdeprecated-declarations]
```

This shows up with the latest `magistrate` and `gtest` @ `1.14.0`.

--- a/tests/unit/test_container.cc
+++ b/tests/unit/test_container.cc
@@ -55,8 +55,8 @@ struct TestContainer : TestHarness { };
 template <typename ContainerT>
 struct TestContainerUnordered : TestHarness { };
 
-TYPED_TEST_CASE_P(TestContainer);
-TYPED_TEST_CASE_P(TestContainerUnordered);
+TYPED_TEST_SUITE_P(TestContainer);
+TYPED_TEST_SUITE_P(TestContainerUnordered);
 
 template <typename T>
 static void testEqualityContainerOrdered(std::queue<T> c1, std::queue<T> t1) {
@@ -170,23 +170,23 @@ using ContUnorderedTypes = ::testing::Types<
   std::unordered_multiset<T>
 >;
 
-REGISTER_TYPED_TEST_CASE_P(TestContainer, test_single_ordered_container);
+REGISTER_TYPED_TEST_SUITE_P(TestContainer, test_single_ordered_container);
 
-INSTANTIATE_TYPED_TEST_CASE_P(Test_int, TestContainer, ContOrderedTypes<int>,);
-INSTANTIATE_TYPED_TEST_CASE_P(Test_int16_t, TestContainer, ContOrderedTypes<int16_t>,);
-INSTANTIATE_TYPED_TEST_CASE_P(Test_int32_t, TestContainer, ContOrderedTypes<int32_t>,);
-INSTANTIATE_TYPED_TEST_CASE_P(Test_int64_t, TestContainer, ContOrderedTypes<int64_t>,);
-INSTANTIATE_TYPED_TEST_CASE_P(Test_double, TestContainer, ContOrderedTypes<double>,);
-INSTANTIATE_TYPED_TEST_CASE_P(Test_float, TestContainer, ContOrderedTypes<float>,);
+INSTANTIATE_TYPED_TEST_SUITE_P(Test_int, TestContainer, ContOrderedTypes<int>,);
+INSTANTIATE_TYPED_TEST_SUITE_P(Test_int16_t, TestContainer, ContOrderedTypes<int16_t>,);
+INSTANTIATE_TYPED_TEST_SUITE_P(Test_int32_t, TestContainer, ContOrderedTypes<int32_t>,);
+INSTANTIATE_TYPED_TEST_SUITE_P(Test_int64_t, TestContainer, ContOrderedTypes<int64_t>,);
+INSTANTIATE_TYPED_TEST_SUITE_P(Test_double, TestContainer, ContOrderedTypes<double>,);
+INSTANTIATE_TYPED_TEST_SUITE_P(Test_float, TestContainer, ContOrderedTypes<float>,);
 
-REGISTER_TYPED_TEST_CASE_P(TestContainerUnordered, test_single_unordered_container);
+REGISTER_TYPED_TEST_SUITE_P(TestContainerUnordered, test_single_unordered_container);
 
-INSTANTIATE_TYPED_TEST_CASE_P(Test_int, TestContainerUnordered, ContUnorderedTypes<int>,);
-INSTANTIATE_TYPED_TEST_CASE_P(Test_int16_t, TestContainerUnordered, ContUnorderedTypes<int16_t>,);
-INSTANTIATE_TYPED_TEST_CASE_P(Test_int32_t, TestContainerUnordered, ContUnorderedTypes<int32_t>,);
-INSTANTIATE_TYPED_TEST_CASE_P(Test_int64_t, TestContainerUnordered, ContUnorderedTypes<int64_t>,);
-INSTANTIATE_TYPED_TEST_CASE_P(Test_double, TestContainerUnordered, ContUnorderedTypes<double>,);
-INSTANTIATE_TYPED_TEST_CASE_P(Test_float, TestContainerUnordered, ContUnorderedTypes<float>,);
+INSTANTIATE_TYPED_TEST_SUITE_P(Test_int, TestContainerUnordered, ContUnorderedTypes<int>,);
+INSTANTIATE_TYPED_TEST_SUITE_P(Test_int16_t, TestContainerUnordered, ContUnorderedTypes<int16_t>,);
+INSTANTIATE_TYPED_TEST_SUITE_P(Test_int32_t, TestContainerUnordered, ContUnorderedTypes<int32_t>,);
+INSTANTIATE_TYPED_TEST_SUITE_P(Test_int64_t, TestContainerUnordered, ContUnorderedTypes<int64_t>,);
+INSTANTIATE_TYPED_TEST_SUITE_P(Test_double, TestContainerUnordered, ContUnorderedTypes<double>,);
+INSTANTIATE_TYPED_TEST_SUITE_P(Test_float, TestContainerUnordered, ContUnorderedTypes<float>,);
 
 template <typename ContainerT>
 struct TestMultiContainer : TestHarness { };
@@ -194,8 +194,8 @@ struct TestMultiContainer : TestHarness { };
 template <typename ContainerT>
 struct TestMultiContainerUnordered : TestHarness { };
 
-TYPED_TEST_CASE_P(TestMultiContainer);
-TYPED_TEST_CASE_P(TestMultiContainerUnordered);
+TYPED_TEST_SUITE_P(TestMultiContainer);
+TYPED_TEST_SUITE_P(TestMultiContainerUnordered);
 
 template <typename ContainerT, typename Pair>
 static void testMultiContainerOrdered(std::initializer_list<Pair> lst) {
@@ -277,30 +277,30 @@ using ContainerMultiTypesUnordered = ::testing::Types<
 template <typename T>
 using ContainerMultiTypesUnorderedDouble = ContainerMultiTypesUnordered<T, double>;
 
-REGISTER_TYPED_TEST_CASE_P(TestMultiContainer, test_multi_container);
-REGISTER_TYPED_TEST_CASE_P(TestMultiContainerUnordered, test_multi_container_unordered);
+REGISTER_TYPED_TEST_SUITE_P(TestMultiContainer, test_multi_container);
+REGISTER_TYPED_TEST_SUITE_P(TestMultiContainerUnordered, test_multi_container_unordered);
 
-INSTANTIATE_TYPED_TEST_CASE_P(TestMulti_int, TestMultiContainer, ContainerMultiTypes<int>, );
-INSTANTIATE_TYPED_TEST_CASE_P(TestMulti_double, TestMultiContainer, ContainerMultiTypes<double>, );
-INSTANTIATE_TYPED_TEST_CASE_P(TestMulti_int64_t, TestMultiContainer, ContainerMultiTypes<int64_t>, );
-INSTANTIATE_TYPED_TEST_CASE_P(TestMulti_int16_t, TestMultiContainer, ContainerMultiTypes<int16_t>, );
-INSTANTIATE_TYPED_TEST_CASE_P(TestMulti_float, TestMultiContainer, ContainerMultiTypes<float>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMulti_int, TestMultiContainer, ContainerMultiTypes<int>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMulti_double, TestMultiContainer, ContainerMultiTypes<double>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMulti_int64_t, TestMultiContainer, ContainerMultiTypes<int64_t>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMulti_int16_t, TestMultiContainer, ContainerMultiTypes<int16_t>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMulti_float, TestMultiContainer, ContainerMultiTypes<float>, );
 
-INSTANTIATE_TYPED_TEST_CASE_P(TestMultiDouble_int, TestMultiContainer, ContainerMultiTypesDouble<int>, );
-INSTANTIATE_TYPED_TEST_CASE_P(TestMultiDouble_int64_t, TestMultiContainer, ContainerMultiTypesDouble<int64_t>, );
-INSTANTIATE_TYPED_TEST_CASE_P(TestMultiDouble_int16_t, TestMultiContainer, ContainerMultiTypesDouble<int16_t>, );
-INSTANTIATE_TYPED_TEST_CASE_P(TestMultiDouble_float, TestMultiContainer, ContainerMultiTypesDouble<float>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMultiDouble_int, TestMultiContainer, ContainerMultiTypesDouble<int>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMultiDouble_int64_t, TestMultiContainer, ContainerMultiTypesDouble<int64_t>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMultiDouble_int16_t, TestMultiContainer, ContainerMultiTypesDouble<int16_t>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMultiDouble_float, TestMultiContainer, ContainerMultiTypesDouble<float>, );
 
 
-INSTANTIATE_TYPED_TEST_CASE_P(TestMulti_int, TestMultiContainerUnordered, ContainerMultiTypesUnordered<int>, );
-INSTANTIATE_TYPED_TEST_CASE_P(TestMulti_double, TestMultiContainerUnordered, ContainerMultiTypesUnordered<double>, );
-INSTANTIATE_TYPED_TEST_CASE_P(TestMulti_int64_t, TestMultiContainerUnordered, ContainerMultiTypesUnordered<int64_t>, );
-INSTANTIATE_TYPED_TEST_CASE_P(TestMulti_int16_t, TestMultiContainerUnordered, ContainerMultiTypesUnordered<int16_t>, );
-INSTANTIATE_TYPED_TEST_CASE_P(TestMulti_float, TestMultiContainerUnordered, ContainerMultiTypesUnordered<float>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMulti_int, TestMultiContainerUnordered, ContainerMultiTypesUnordered<int>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMulti_double, TestMultiContainerUnordered, ContainerMultiTypesUnordered<double>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMulti_int64_t, TestMultiContainerUnordered, ContainerMultiTypesUnordered<int64_t>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMulti_int16_t, TestMultiContainerUnordered, ContainerMultiTypesUnordered<int16_t>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMulti_float, TestMultiContainerUnordered, ContainerMultiTypesUnordered<float>, );
 
-INSTANTIATE_TYPED_TEST_CASE_P(TestMultiDouble_int, TestMultiContainerUnordered, ContainerMultiTypesUnorderedDouble<int>, );
-INSTANTIATE_TYPED_TEST_CASE_P(TestMultiDouble_int64_t, TestMultiContainerUnordered, ContainerMultiTypesUnorderedDouble<int64_t>, );
-INSTANTIATE_TYPED_TEST_CASE_P(TestMultiDouble_int16_t, TestMultiContainerUnordered, ContainerMultiTypesUnorderedDouble<int16_t>, );
-INSTANTIATE_TYPED_TEST_CASE_P(TestMultiDouble_float, TestMultiContainerUnordered, ContainerMultiTypesUnorderedDouble<float>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMultiDouble_int, TestMultiContainerUnordered, ContainerMultiTypesUnorderedDouble<int>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMultiDouble_int64_t, TestMultiContainerUnordered, ContainerMultiTypesUnorderedDouble<int64_t>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMultiDouble_int16_t, TestMultiContainerUnordered, ContainerMultiTypesUnorderedDouble<int16_t>, );
+INSTANTIATE_TYPED_TEST_SUITE_P(TestMultiDouble_float, TestMultiContainerUnordered, ContainerMultiTypesUnorderedDouble<float>, );
 
 }}} // end namespace checkpoint::tests::unit

--- a/tests/unit/test_kokkos_3d_commons.h
+++ b/tests/unit/test_kokkos_3d_commons.h
@@ -163,7 +163,7 @@ using Test3DConstTypesStride =
 
 template <typename ParamT> struct KokkosViewTest3D : KokkosViewTest<ParamT> { };
 
-TYPED_TEST_CASE_P(KokkosViewTest3D);
+TYPED_TEST_SUITE_P(KokkosViewTest3D);
 
 TYPED_TEST_P(KokkosViewTest3D, test_3d_any) {
   using namespace checkpoint;

--- a/tests/unit/test_kokkos_serialize_0d.cc
+++ b/tests/unit/test_kokkos_serialize_0d.cc
@@ -48,7 +48,7 @@
 
 template <typename ParamT> struct KokkosViewTest0D : KokkosViewTest<ParamT> { };
 
-TYPED_TEST_CASE_P(KokkosViewTest0D);
+TYPED_TEST_SUITE_P(KokkosViewTest0D);
 
 TYPED_TEST_P(KokkosViewTest0D, test_0d_any) {
   using namespace checkpoint;
@@ -72,11 +72,11 @@ TYPED_TEST_P(KokkosViewTest0D, test_0d_any) {
   }
 }
 
-REGISTER_TYPED_TEST_CASE_P(KokkosViewTest0D, test_0d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosViewTest0D, test_0d_any);
 
 #if DO_UNIT_TESTS_FOR_VIEW
 
-INSTANTIATE_TYPED_TEST_CASE_P(test_0d, KokkosViewTest0D, Test0DTypes, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_0d, KokkosViewTest0D, Test0DTypes, );
 
 #endif
 

--- a/tests/unit/test_kokkos_serialize_1d.cc
+++ b/tests/unit/test_kokkos_serialize_1d.cc
@@ -47,7 +47,7 @@
 
 template <typename ParamT> struct KokkosViewTest1D : KokkosViewTest<ParamT> { };
 
-TYPED_TEST_CASE_P(KokkosViewTest1D);
+TYPED_TEST_SUITE_P(KokkosViewTest1D);
 
 TYPED_TEST_P(KokkosViewTest1D, test_1d_any) {
   using namespace checkpoint;
@@ -76,16 +76,16 @@ TYPED_TEST_P(KokkosViewTest1D, test_1d_any) {
 }
 
 
-REGISTER_TYPED_TEST_CASE_P(KokkosViewTest1D, test_1d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosViewTest1D, test_1d_any);
 
 #if DO_UNIT_TESTS_FOR_VIEW
 
-INSTANTIATE_TYPED_TEST_CASE_P(test_1d_L, KokkosViewTest1D, Test1DTypesLeft,);
-INSTANTIATE_TYPED_TEST_CASE_P(test_1d_R, KokkosViewTest1D, Test1DTypesRight,);
-INSTANTIATE_TYPED_TEST_CASE_P(test_1d_S, KokkosViewTest1D, Test1DTypesStride,);
-INSTANTIATE_TYPED_TEST_CASE_P(test_1d_L_C, KokkosViewTest1D, Test1DConstTypesLeft,);
-INSTANTIATE_TYPED_TEST_CASE_P(test_1d_R_C, KokkosViewTest1D, Test1DConstTypesRight,);
-INSTANTIATE_TYPED_TEST_CASE_P(test_1d_S_C, KokkosViewTest1D, Test1DConstTypesStride,);
+INSTANTIATE_TYPED_TEST_SUITE_P(test_1d_L, KokkosViewTest1D, Test1DTypesLeft,);
+INSTANTIATE_TYPED_TEST_SUITE_P(test_1d_R, KokkosViewTest1D, Test1DTypesRight,);
+INSTANTIATE_TYPED_TEST_SUITE_P(test_1d_S, KokkosViewTest1D, Test1DTypesStride,);
+INSTANTIATE_TYPED_TEST_SUITE_P(test_1d_L_C, KokkosViewTest1D, Test1DConstTypesLeft,);
+INSTANTIATE_TYPED_TEST_SUITE_P(test_1d_R_C, KokkosViewTest1D, Test1DConstTypesRight,);
+INSTANTIATE_TYPED_TEST_SUITE_P(test_1d_S_C, KokkosViewTest1D, Test1DConstTypesStride,);
 
 #endif
 
@@ -96,7 +96,7 @@ INSTANTIATE_TYPED_TEST_CASE_P(test_1d_S_C, KokkosViewTest1D, Test1DConstTypesStr
 template <typename ParamT>
 struct KokkosDynamicViewTest : KokkosViewTest<ParamT> { };
 
-TYPED_TEST_CASE_P(KokkosDynamicViewTest);
+TYPED_TEST_SUITE_P(KokkosDynamicViewTest);
 
 TYPED_TEST_P(KokkosDynamicViewTest, test_dynamic_1d) {
   using namespace checkpoint;
@@ -124,9 +124,9 @@ TYPED_TEST_P(KokkosDynamicViewTest, test_dynamic_1d) {
   serializeAny<ViewType>(in_view, &compare1dDynamic<ViewType>);
 }
 
-REGISTER_TYPED_TEST_CASE_P(KokkosDynamicViewTest, test_dynamic_1d);
+REGISTER_TYPED_TEST_SUITE_P(KokkosDynamicViewTest, test_dynamic_1d);
 
-INSTANTIATE_TYPED_TEST_CASE_P(
+INSTANTIATE_TYPED_TEST_SUITE_P(
   test_dynamic_view_1, KokkosDynamicViewTest, DynamicTestTypes,
 );
 

--- a/tests/unit/test_kokkos_serialize_2d.cc
+++ b/tests/unit/test_kokkos_serialize_2d.cc
@@ -49,7 +49,7 @@
 
 template <typename ParamT> struct KokkosViewTest2D : KokkosViewTest<ParamT> { };
 
-TYPED_TEST_CASE_P(KokkosViewTest2D);
+TYPED_TEST_SUITE_P(KokkosViewTest2D);
 
 TYPED_TEST_P(KokkosViewTest2D, test_2d_any) {
   using namespace checkpoint;
@@ -77,16 +77,16 @@ TYPED_TEST_P(KokkosViewTest2D, test_2d_any) {
   }
 }
 
-REGISTER_TYPED_TEST_CASE_P(KokkosViewTest2D, test_2d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosViewTest2D, test_2d_any);
 
 #if DO_UNIT_TESTS_FOR_VIEW
 
-INSTANTIATE_TYPED_TEST_CASE_P(test_2d_L,   KokkosViewTest2D, Test2DTypesLeft, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_2d_R,   KokkosViewTest2D, Test2DTypesRight, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_2d_S,   KokkosViewTest2D, Test2DTypesStride, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_2d_L_C, KokkosViewTest2D, Test2DConstTypesLeft, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_2d_R_C, KokkosViewTest2D, Test2DConstTypesRight, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_2d_S_C, KokkosViewTest2D, Test2DConstTypesStride, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_2d_L,   KokkosViewTest2D, Test2DTypesLeft, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_2d_R,   KokkosViewTest2D, Test2DTypesRight, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_2d_S,   KokkosViewTest2D, Test2DTypesStride, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_2d_L_C, KokkosViewTest2D, Test2DConstTypesLeft, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_2d_R_C, KokkosViewTest2D, Test2DConstTypesRight, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_2d_S_C, KokkosViewTest2D, Test2DConstTypesStride, );
 
 #endif
 #endif

--- a/tests/unit/test_kokkos_serialize_3d_left.cc
+++ b/tests/unit/test_kokkos_serialize_3d_left.cc
@@ -47,9 +47,9 @@
 #include "test_kokkos_3d_commons.h"
 
 #if DO_UNIT_TESTS_FOR_VIEW
-REGISTER_TYPED_TEST_CASE_P(KokkosViewTest3D, test_3d_any);
-INSTANTIATE_TYPED_TEST_CASE_P(test_3d_L, KokkosViewTest3D, Test3DTypesLeft, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_3d_L_C, KokkosViewTest3D, Test3DConstTypesLeft, );
+REGISTER_TYPED_TEST_SUITE_P(KokkosViewTest3D, test_3d_any);
+INSTANTIATE_TYPED_TEST_SUITE_P(test_3d_L, KokkosViewTest3D, Test3DTypesLeft, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_3d_L_C, KokkosViewTest3D, Test3DConstTypesLeft, );
 #endif
 
 #endif

--- a/tests/unit/test_kokkos_serialize_3d_right.cc
+++ b/tests/unit/test_kokkos_serialize_3d_right.cc
@@ -47,9 +47,9 @@
 #include "test_kokkos_3d_commons.h"
 
 #if DO_UNIT_TESTS_FOR_VIEW
-REGISTER_TYPED_TEST_CASE_P(KokkosViewTest3D, test_3d_any);
-INSTANTIATE_TYPED_TEST_CASE_P(test_3d_R, KokkosViewTest3D, Test3DTypesRight, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_3d_R_C, KokkosViewTest3D, Test3DConstTypesRight, );
+REGISTER_TYPED_TEST_SUITE_P(KokkosViewTest3D, test_3d_any);
+INSTANTIATE_TYPED_TEST_SUITE_P(test_3d_R, KokkosViewTest3D, Test3DTypesRight, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_3d_R_C, KokkosViewTest3D, Test3DConstTypesRight, );
 #endif
 
 #endif

--- a/tests/unit/test_kokkos_serialize_3d_stride.cc
+++ b/tests/unit/test_kokkos_serialize_3d_stride.cc
@@ -47,9 +47,9 @@
 #include "test_kokkos_3d_commons.h"
 
 #if DO_UNIT_TESTS_FOR_VIEW
-REGISTER_TYPED_TEST_CASE_P(KokkosViewTest3D, test_3d_any);
-INSTANTIATE_TYPED_TEST_CASE_P(test_3d_S, KokkosViewTest3D, Test3DTypesStride, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_3d_S_C, KokkosViewTest3D, Test3DConstTypesStride, );
+REGISTER_TYPED_TEST_SUITE_P(KokkosViewTest3D, test_3d_any);
+INSTANTIATE_TYPED_TEST_SUITE_P(test_3d_S, KokkosViewTest3D, Test3DTypesStride, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_3d_S_C, KokkosViewTest3D, Test3DConstTypesStride, );
 #endif
 
 #endif

--- a/tests/unit/test_kokkos_serialize_dynrankview.cc
+++ b/tests/unit/test_kokkos_serialize_dynrankview.cc
@@ -66,11 +66,11 @@ struct KokkosDynRankViewTest2D : KokkosViewTest<ParamT> { };
 template <typename ParamT>
 struct KokkosDynRankViewTest3D : KokkosViewTest<ParamT> { };
 
-TYPED_TEST_CASE_P(KokkosDynRankViewTestEmpty);
-TYPED_TEST_CASE_P(KokkosDynRankViewTest0D);
-TYPED_TEST_CASE_P(KokkosDynRankViewTest1D);
-TYPED_TEST_CASE_P(KokkosDynRankViewTest2D);
-TYPED_TEST_CASE_P(KokkosDynRankViewTest3D);
+TYPED_TEST_SUITE_P(KokkosDynRankViewTestEmpty);
+TYPED_TEST_SUITE_P(KokkosDynRankViewTest0D);
+TYPED_TEST_SUITE_P(KokkosDynRankViewTest1D);
+TYPED_TEST_SUITE_P(KokkosDynRankViewTest2D);
+TYPED_TEST_SUITE_P(KokkosDynRankViewTest3D);
 
 TYPED_TEST_P(KokkosDynRankViewTestEmpty, test_empty_any) {
   using namespace checkpoint;
@@ -149,19 +149,19 @@ TYPED_TEST_P(KokkosDynRankViewTest3D, test_3d_any) {
   EXPECT_EQ(out_view->rank(), unsigned(3));
 }
 
-REGISTER_TYPED_TEST_CASE_P(KokkosDynRankViewTestEmpty, test_empty_any);
-REGISTER_TYPED_TEST_CASE_P(KokkosDynRankViewTest0D, test_0d_any);
-REGISTER_TYPED_TEST_CASE_P(KokkosDynRankViewTest1D, test_1d_any);
-REGISTER_TYPED_TEST_CASE_P(KokkosDynRankViewTest2D, test_2d_any);
-REGISTER_TYPED_TEST_CASE_P(KokkosDynRankViewTest3D, test_3d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosDynRankViewTestEmpty, test_empty_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosDynRankViewTest0D, test_0d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosDynRankViewTest1D, test_1d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosDynRankViewTest2D, test_2d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosDynRankViewTest3D, test_3d_any);
 
 #if DO_UNIT_TESTS_FOR_VIEW
 
-INSTANTIATE_TYPED_TEST_CASE_P(test_dynrank_empty , KokkosDynRankViewTestEmpty, DynRankViewTestTypes, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_dynrank_0, KokkosDynRankViewTest0D, DynRankViewTestTypes, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_dynrank_1, KokkosDynRankViewTest1D, DynRankViewTestTypes, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_dynrank_2, KokkosDynRankViewTest2D, DynRankViewTestTypes, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_dynrank_3, KokkosDynRankViewTest3D, DynRankViewTestTypes, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_dynrank_empty , KokkosDynRankViewTestEmpty, DynRankViewTestTypes, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_dynrank_0, KokkosDynRankViewTest0D, DynRankViewTestTypes, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_dynrank_1, KokkosDynRankViewTest1D, DynRankViewTestTypes, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_dynrank_2, KokkosDynRankViewTest2D, DynRankViewTestTypes, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_dynrank_3, KokkosDynRankViewTest3D, DynRankViewTestTypes, );
 
 #endif
 

--- a/tests/unit/test_reconstruct.cc
+++ b/tests/unit/test_reconstruct.cc
@@ -55,7 +55,7 @@ namespace checkpoint { namespace tests { namespace unit {
 template <typename T>
 struct TestReconstruct : TestHarness { };
 
-TYPED_TEST_CASE_P(TestReconstruct);
+TYPED_TEST_SUITE_P(TestReconstruct);
 
 static constexpr int const u_val = 43;
 static constexpr int const w_val = 38;
@@ -261,7 +261,7 @@ using ConstructTypes = ::testing::Types<
   UserObjectE
 >;
 
-REGISTER_TYPED_TEST_CASE_P(TestReconstruct, test_reconstruct_multi_type);
-INSTANTIATE_TYPED_TEST_CASE_P(Test_reconstruct, TestReconstruct, ConstructTypes, );
+REGISTER_TYPED_TEST_SUITE_P(TestReconstruct, test_reconstruct_multi_type);
+INSTANTIATE_TYPED_TEST_SUITE_P(Test_reconstruct, TestReconstruct, ConstructTypes, );
 
 }}} // end namespace checkpoint::tests::unit

--- a/tests/unit/test_serialize_file.cc
+++ b/tests/unit/test_serialize_file.cc
@@ -57,8 +57,8 @@ struct TestSerializeFile : TestHarness { };
 template <typename T>
 struct TestSerializeFileInPlace : TestHarness { };
 
-TYPED_TEST_CASE_P(TestSerializeFile);
-TYPED_TEST_CASE_P(TestSerializeFileInPlace);
+TYPED_TEST_SUITE_P(TestSerializeFile);
+TYPED_TEST_SUITE_P(TestSerializeFileInPlace);
 
 static constexpr int const u_val = 934;
 
@@ -161,10 +161,10 @@ using ConstructTypes = ::testing::Types<
   UserObjectC
 >;
 
-REGISTER_TYPED_TEST_CASE_P(TestSerializeFile, test_serialize_file_multi);
-REGISTER_TYPED_TEST_CASE_P(TestSerializeFileInPlace, test_serialize_file_multi_in_place);
+REGISTER_TYPED_TEST_SUITE_P(TestSerializeFile, test_serialize_file_multi);
+REGISTER_TYPED_TEST_SUITE_P(TestSerializeFileInPlace, test_serialize_file_multi_in_place);
 
-INSTANTIATE_TYPED_TEST_CASE_P(test_file, TestSerializeFile, ConstructTypes, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_file_in_place, TestSerializeFileInPlace, ConstructTypes, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_file, TestSerializeFile, ConstructTypes, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_file_in_place, TestSerializeFileInPlace, ConstructTypes, );
 
 }}} // end namespace checkpoint::tests::unit

--- a/tests/unit/test_serialize_stream.cc
+++ b/tests/unit/test_serialize_stream.cc
@@ -58,8 +58,8 @@ struct TestSerializeStream : TestHarness { };
 template <typename T>
 struct TestSerializeStreamInPlace : TestHarness { };
 
-TYPED_TEST_CASE_P(TestSerializeStream);
-TYPED_TEST_CASE_P(TestSerializeStreamInPlace);
+TYPED_TEST_SUITE_P(TestSerializeStream);
+TYPED_TEST_SUITE_P(TestSerializeStreamInPlace);
 
 static constexpr int const u_val = 934;
 
@@ -176,10 +176,10 @@ using ConstructTypes = ::testing::Types<
   UserObjectC
 >;
 
-REGISTER_TYPED_TEST_CASE_P(TestSerializeStream, test_serialize_stream_multi);
-REGISTER_TYPED_TEST_CASE_P(TestSerializeStreamInPlace, test_serialize_stream_multi_in_place);
+REGISTER_TYPED_TEST_SUITE_P(TestSerializeStream, test_serialize_stream_multi);
+REGISTER_TYPED_TEST_SUITE_P(TestSerializeStreamInPlace, test_serialize_stream_multi_in_place);
 
-INSTANTIATE_TYPED_TEST_CASE_P(test_file, TestSerializeStream, ConstructTypes, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_file_in_place, TestSerializeStreamInPlace, ConstructTypes, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_file, TestSerializeStream, ConstructTypes, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_file_in_place, TestSerializeStreamInPlace, ConstructTypes, );
 
 }}} // end namespace checkpoint::tests::unit

--- a/tests/unit/test_tagged_construct.cc
+++ b/tests/unit/test_tagged_construct.cc
@@ -55,7 +55,7 @@ namespace checkpoint { namespace tests { namespace unit {
 template <typename T>
 struct TestTaggedConstruct : TestHarness { };
 
-TYPED_TEST_CASE_P(TestTaggedConstruct);
+TYPED_TEST_SUITE_P(TestTaggedConstruct);
 
 static constexpr int const u_val = 43;
 
@@ -132,7 +132,7 @@ using ConstructTypes = ::testing::Types<
   UserObjectA, UserObjectB
 >;
 
-REGISTER_TYPED_TEST_CASE_P(TestTaggedConstruct, test_tagged_construct);
-INSTANTIATE_TYPED_TEST_CASE_P(test_tagged, TestTaggedConstruct, ConstructTypes, );
+REGISTER_TYPED_TEST_SUITE_P(TestTaggedConstruct, test_tagged_construct);
+INSTANTIATE_TYPED_TEST_SUITE_P(test_tagged, TestTaggedConstruct, ConstructTypes, );
 
 }}} // end namespace checkpoint::tests::unit

--- a/tests/unit/test_user_traits.cc
+++ b/tests/unit/test_user_traits.cc
@@ -53,7 +53,7 @@
 namespace checkpoint { namespace tests { namespace unit {
 template <typename T>
 struct TestUserTraits : TestHarness { };
-TYPED_TEST_CASE_P(TestUserTraits);
+TYPED_TEST_SUITE_P(TestUserTraits);
 
 struct CheckpointTrait {};
 struct CheckpointTraitNonintrusive {};
@@ -138,9 +138,9 @@ TYPED_TEST_P(TestUserTraits, test_serialize_extra) {
   EXPECT_EQ(objA.name, old_name);
   EXPECT_NE(objA.name, new_name);
 }
-REGISTER_TYPED_TEST_CASE_P(TestUserTraits, test_serialize_extra);
+REGISTER_TYPED_TEST_SUITE_P(TestUserTraits, test_serialize_extra);
 using TraitTypes = ::testing::Types<CheckpointTrait, CheckpointTraitNonintrusive, CheckpointNamespace::CheckpointTraitNamespaced>;
-INSTANTIATE_TYPED_TEST_CASE_P(checkpoint_trait, TestUserTraits, TraitTypes, );
+INSTANTIATE_TYPED_TEST_SUITE_P(checkpoint_trait, TestUserTraits, TraitTypes, );
 
 
 

--- a/tests/unit/test_virtual_serialize.cc
+++ b/tests/unit/test_virtual_serialize.cc
@@ -581,7 +581,7 @@ struct TestWrapper {
 template <typename T>
 struct TestVirtualSerialize : TestHarness { };
 
-TYPED_TEST_CASE_P(TestVirtualSerialize);
+TYPED_TEST_SUITE_P(TestVirtualSerialize);
 
 /*
  * General test of serialization/deserialization for input object types
@@ -605,9 +605,9 @@ using ConstructTypes = ::testing::Types<
   test_3::TestWrapper
 >;
 
-REGISTER_TYPED_TEST_CASE_P(TestVirtualSerialize, test_virtual_serialize);
+REGISTER_TYPED_TEST_SUITE_P(TestVirtualSerialize, test_virtual_serialize);
 
-INSTANTIATE_TYPED_TEST_CASE_P(
+INSTANTIATE_TYPED_TEST_SUITE_P(
   test_virtual_serialize_inst, TestVirtualSerialize, ConstructTypes,
 );
 

--- a/tests/unit/tests_mpi/test_kokkos_serialize_0d_mpi.cc
+++ b/tests/unit/tests_mpi/test_kokkos_serialize_0d_mpi.cc
@@ -46,7 +46,7 @@
 #include "tests_mpi/test_commons_mpi.h"
 
 template <typename ParamT> struct KokkosViewTest0DMPI : KokkosViewTest<ParamT> { };
-TYPED_TEST_CASE_P(KokkosViewTest0DMPI);
+TYPED_TEST_SUITE_P(KokkosViewTest0DMPI);
 
 TYPED_TEST_P(KokkosViewTest0DMPI, test_0d_any) {
   using namespace checkpoint;
@@ -71,11 +71,11 @@ TYPED_TEST_P(KokkosViewTest0DMPI, test_0d_any) {
   }
 }
 
-REGISTER_TYPED_TEST_CASE_P(KokkosViewTest0DMPI, test_0d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosViewTest0DMPI, test_0d_any);
 
 #if DO_UNIT_TESTS_FOR_VIEW
 
-INSTANTIATE_TYPED_TEST_CASE_P(test_0d, KokkosViewTest0DMPI, Test0DTypes, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_0d, KokkosViewTest0DMPI, Test0DTypes, );
 
 #endif
 #endif

--- a/tests/unit/tests_mpi/test_kokkos_serialize_1d_mpi.cc
+++ b/tests/unit/tests_mpi/test_kokkos_serialize_1d_mpi.cc
@@ -47,7 +47,7 @@
 
 template <typename ParamT> struct KokkosViewTest1DMPI : KokkosViewTest<ParamT> { };
 
-TYPED_TEST_CASE_P(KokkosViewTest1DMPI);
+TYPED_TEST_SUITE_P(KokkosViewTest1DMPI);
 
 TYPED_TEST_P(KokkosViewTest1DMPI, test_1d_any) {
   using namespace checkpoint;
@@ -77,16 +77,16 @@ TYPED_TEST_P(KokkosViewTest1DMPI, test_1d_any) {
 }
 
 
-REGISTER_TYPED_TEST_CASE_P(KokkosViewTest1DMPI, test_1d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosViewTest1DMPI, test_1d_any);
 
 #if DO_UNIT_TESTS_FOR_VIEW
 
-INSTANTIATE_TYPED_TEST_CASE_P(test_1d_L, KokkosViewTest1DMPI, Test1DTypesLeft, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_1d_R, KokkosViewTest1DMPI, Test1DTypesRight, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_1d_S, KokkosViewTest1DMPI, Test1DTypesStride, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_1d_L_C, KokkosViewTest1DMPI, Test1DConstTypesLeft, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_1d_R_C, KokkosViewTest1DMPI, Test1DConstTypesRight, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_1d_S_C, KokkosViewTest1DMPI, Test1DConstTypesStride, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_1d_L, KokkosViewTest1DMPI, Test1DTypesLeft, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_1d_R, KokkosViewTest1DMPI, Test1DTypesRight, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_1d_S, KokkosViewTest1DMPI, Test1DTypesStride, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_1d_L_C, KokkosViewTest1DMPI, Test1DConstTypesLeft, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_1d_R_C, KokkosViewTest1DMPI, Test1DConstTypesRight, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_1d_S_C, KokkosViewTest1DMPI, Test1DConstTypesStride, );
 
 #endif
 
@@ -97,7 +97,7 @@ INSTANTIATE_TYPED_TEST_CASE_P(test_1d_S_C, KokkosViewTest1DMPI, Test1DConstTypes
 template <typename ParamT>
 struct KokkosDynamicViewTestMPI : KokkosViewTest<ParamT> { };
 
-TYPED_TEST_CASE_P(KokkosDynamicViewTestMPI);
+TYPED_TEST_SUITE_P(KokkosDynamicViewTestMPI);
 
 TYPED_TEST_P(KokkosDynamicViewTestMPI, test_dynamic_1d) {
   using namespace checkpoint;
@@ -117,10 +117,10 @@ TYPED_TEST_P(KokkosDynamicViewTestMPI, test_dynamic_1d) {
   serializeAnyMPI<ViewType>(in_view, &compare1dDynamic<ViewType>);
 }
 
-REGISTER_TYPED_TEST_CASE_P(KokkosDynamicViewTestMPI, test_dynamic_1d);
+REGISTER_TYPED_TEST_SUITE_P(KokkosDynamicViewTestMPI, test_dynamic_1d);
 
 #if DO_UNIT_TESTS_FOR_VIEW
-INSTANTIATE_TYPED_TEST_CASE_P(
+INSTANTIATE_TYPED_TEST_SUITE_P(
   test_dynamic_view_1, KokkosDynamicViewTestMPI, DynamicTestTypes,
 );
 #endif

--- a/tests/unit/tests_mpi/test_kokkos_serialize_2d_mpi_left.cc
+++ b/tests/unit/tests_mpi/test_kokkos_serialize_2d_mpi_left.cc
@@ -47,7 +47,7 @@
 
 template <typename ParamT> struct KokkosViewTest2DMPI : KokkosViewTest<ParamT> { };
 
-TYPED_TEST_CASE_P(KokkosViewTest2DMPI);
+TYPED_TEST_SUITE_P(KokkosViewTest2DMPI);
 
 TYPED_TEST_P(KokkosViewTest2DMPI, test_2d_any) {
   using namespace checkpoint;
@@ -75,12 +75,12 @@ TYPED_TEST_P(KokkosViewTest2DMPI, test_2d_any) {
   }
 }
 
-REGISTER_TYPED_TEST_CASE_P(KokkosViewTest2DMPI, test_2d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosViewTest2DMPI, test_2d_any);
 
 #if DO_UNIT_TESTS_FOR_VIEW
 
-INSTANTIATE_TYPED_TEST_CASE_P(test_2d_L,   KokkosViewTest2DMPI, Test2DTypesLeft, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_2d_L_C, KokkosViewTest2DMPI, Test2DConstTypesLeft, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_2d_L,   KokkosViewTest2DMPI, Test2DTypesLeft, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_2d_L_C, KokkosViewTest2DMPI, Test2DConstTypesLeft, );
 
 #endif
 #endif

--- a/tests/unit/tests_mpi/test_kokkos_serialize_2d_mpi_right.cc
+++ b/tests/unit/tests_mpi/test_kokkos_serialize_2d_mpi_right.cc
@@ -47,7 +47,7 @@
 
 template <typename ParamT> struct KokkosViewTest2DMPI : KokkosViewTest<ParamT> { };
 
-TYPED_TEST_CASE_P(KokkosViewTest2DMPI);
+TYPED_TEST_SUITE_P(KokkosViewTest2DMPI);
 
 TYPED_TEST_P(KokkosViewTest2DMPI, test_2d_any) {
   using namespace checkpoint;
@@ -75,12 +75,12 @@ TYPED_TEST_P(KokkosViewTest2DMPI, test_2d_any) {
   }
 }
 
-REGISTER_TYPED_TEST_CASE_P(KokkosViewTest2DMPI, test_2d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosViewTest2DMPI, test_2d_any);
 
 #if DO_UNIT_TESTS_FOR_VIEW
 
-INSTANTIATE_TYPED_TEST_CASE_P(test_2d_R,   KokkosViewTest2DMPI, Test2DTypesRight, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_2d_R_C, KokkosViewTest2DMPI, Test2DConstTypesRight, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_2d_R,   KokkosViewTest2DMPI, Test2DTypesRight, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_2d_R_C, KokkosViewTest2DMPI, Test2DConstTypesRight, );
 
 #endif
 #endif

--- a/tests/unit/tests_mpi/test_kokkos_serialize_2d_mpi_stride.cc
+++ b/tests/unit/tests_mpi/test_kokkos_serialize_2d_mpi_stride.cc
@@ -47,7 +47,7 @@
 
 template <typename ParamT> struct KokkosViewTest2DMPI : KokkosViewTest<ParamT> { };
 
-TYPED_TEST_CASE_P(KokkosViewTest2DMPI);
+TYPED_TEST_SUITE_P(KokkosViewTest2DMPI);
 
 TYPED_TEST_P(KokkosViewTest2DMPI, test_2d_any) {
   using namespace checkpoint;
@@ -75,12 +75,12 @@ TYPED_TEST_P(KokkosViewTest2DMPI, test_2d_any) {
   }
 }
 
-REGISTER_TYPED_TEST_CASE_P(KokkosViewTest2DMPI, test_2d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosViewTest2DMPI, test_2d_any);
 
 #if DO_UNIT_TESTS_FOR_VIEW
 
-INSTANTIATE_TYPED_TEST_CASE_P(test_2d_S,   KokkosViewTest2DMPI, Test2DTypesStride, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_2d_S_C, KokkosViewTest2DMPI, Test2DConstTypesStride, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_2d_S,   KokkosViewTest2DMPI, Test2DTypesStride, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_2d_S_C, KokkosViewTest2DMPI, Test2DConstTypesStride, );
 
 #endif
 #endif

--- a/tests/unit/tests_mpi/test_kokkos_serialize_3d_mpi_left.cc
+++ b/tests/unit/tests_mpi/test_kokkos_serialize_3d_mpi_left.cc
@@ -49,7 +49,7 @@
 template <typename ParamT>
 struct KokkosViewTest3DMPI : KokkosViewTest<ParamT> { };
 
-TYPED_TEST_CASE_P(KokkosViewTest3DMPI);
+TYPED_TEST_SUITE_P(KokkosViewTest3DMPI);
 
 TYPED_TEST_P(KokkosViewTest3DMPI, test_3d_any) {
   using namespace checkpoint;
@@ -79,12 +79,12 @@ TYPED_TEST_P(KokkosViewTest3DMPI, test_3d_any) {
   }
 }
 
-REGISTER_TYPED_TEST_CASE_P(KokkosViewTest3DMPI, test_3d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosViewTest3DMPI, test_3d_any);
 
 #if DO_UNIT_TESTS_FOR_VIEW
 
-INSTANTIATE_TYPED_TEST_CASE_P(test_3d_L, KokkosViewTest3DMPI, Test3DTypesLeft, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_3d_L_C, KokkosViewTest3DMPI, Test3DConstTypesLeft, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_3d_L, KokkosViewTest3DMPI, Test3DTypesLeft, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_3d_L_C, KokkosViewTest3DMPI, Test3DConstTypesLeft, );
 
 #endif
 

--- a/tests/unit/tests_mpi/test_kokkos_serialize_3d_mpi_right.cc
+++ b/tests/unit/tests_mpi/test_kokkos_serialize_3d_mpi_right.cc
@@ -49,7 +49,7 @@
 template <typename ParamT>
 struct KokkosViewTest3DMPI : KokkosViewTest<ParamT> { };
 
-TYPED_TEST_CASE_P(KokkosViewTest3DMPI);
+TYPED_TEST_SUITE_P(KokkosViewTest3DMPI);
 
 TYPED_TEST_P(KokkosViewTest3DMPI, test_3d_any) {
   using namespace checkpoint;
@@ -79,12 +79,12 @@ TYPED_TEST_P(KokkosViewTest3DMPI, test_3d_any) {
   }
 }
 
-REGISTER_TYPED_TEST_CASE_P(KokkosViewTest3DMPI, test_3d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosViewTest3DMPI, test_3d_any);
 
 #if DO_UNIT_TESTS_FOR_VIEW
 
-INSTANTIATE_TYPED_TEST_CASE_P(test_3d_R, KokkosViewTest3DMPI, Test3DTypesRight, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_3d_R_C, KokkosViewTest3DMPI, Test3DConstTypesRight, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_3d_R, KokkosViewTest3DMPI, Test3DTypesRight, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_3d_R_C, KokkosViewTest3DMPI, Test3DConstTypesRight, );
 
 #endif
 

--- a/tests/unit/tests_mpi/test_kokkos_serialize_3d_mpi_stride.cc
+++ b/tests/unit/tests_mpi/test_kokkos_serialize_3d_mpi_stride.cc
@@ -49,7 +49,7 @@
 template <typename ParamT>
 struct KokkosViewTest3DMPI : KokkosViewTest<ParamT> { };
 
-TYPED_TEST_CASE_P(KokkosViewTest3DMPI);
+TYPED_TEST_SUITE_P(KokkosViewTest3DMPI);
 
 TYPED_TEST_P(KokkosViewTest3DMPI, test_3d_any) {
   using namespace checkpoint;
@@ -79,12 +79,12 @@ TYPED_TEST_P(KokkosViewTest3DMPI, test_3d_any) {
   }
 }
 
-REGISTER_TYPED_TEST_CASE_P(KokkosViewTest3DMPI, test_3d_any);
+REGISTER_TYPED_TEST_SUITE_P(KokkosViewTest3DMPI, test_3d_any);
 
 #if DO_UNIT_TESTS_FOR_VIEW
 
-INSTANTIATE_TYPED_TEST_CASE_P(test_3d_S, KokkosViewTest3DMPI, Test3DTypesStride, );
-INSTANTIATE_TYPED_TEST_CASE_P(test_3d_S_C, KokkosViewTest3DMPI, Test3DConstTypesStride, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_3d_S, KokkosViewTest3DMPI, Test3DTypesStride, );
+INSTANTIATE_TYPED_TEST_SUITE_P(test_3d_S_C, KokkosViewTest3DMPI, Test3DConstTypesStride, );
 
 #endif
 

